services/galley/src/Galley/Data.hs

```diff
@@ -381,8 +393,9 @@ createConversation :: UserId
                    -> Maybe ConvTeamInfo
                    -> Maybe Milliseconds                  -- ^ Message timer
                    -> Maybe ReceiptMode
+                   -> RoleName
                    -> Galley Conversation
-createConversation usr name acc role others tinfo mtimer recpt = do
+createConversation usr name acc role others tinfo mtimer recpt othersRole = do
```

Comment:
`othersRole`? `conversationRole` ?

Reply:
Right, I renamed it to try and be more explicit; it refers to the role that `others` should have: 616ff0edeb353a1018efd268a5fe0d7c07f468ca